packages/components/src/tokens/colors/index.scss

```diff
@@ -30,6 +30,9 @@ $colors: (
   "black": (
     500: #000,
   ),
+  "transparentBlack": (
+    500: rgba(0, 0, 0, 0.5),
```

Comment:
hmm ja by som mozno preferoval nieco take ze "transparentBlack" bolo sucastou "black" - nie takto samostatne, alebo to tu nemal vobec v tokenoch ako to nie je esencialna farba. Da sa to nejak inak skombonovat, ze sa pouzije `color('black')` s nejakou funkciou, aby sme dostali semitransparentu?